pkg/operator/encryption/controllers.go

```diff
@@ -65,6 +65,7 @@ func NewControllers(
 	return &Controllers{
 		controllers: []runner{
 			controllers.NewKeyController(
+				"EncryptionKeyController",
```

Comment:
same questions about the instance names here.

Reply:
Fixed all of them